test/ng/compileSpec.js

```diff
@@ -2762,6 +2762,23 @@ describe('$compile', function() {
     }));
 
 
+    it('should handle consecutive text elements as a single text element', inject(function($rootScope, $compile) {
+      window.addMutationObserver = function() {
```

Comment:
@lgalfaso, I noticed that the added test would pass on IE11 even without the fix.
Looking into it, it turned out I (unwittingly) tricked you into changing `subtree: true` to `sublist: true`, which effectively makes the test useless :disappointed: 

Submitted a fix in #12061.